app/client/ui/WebhookPage.ts

```diff
@@ -107,17 +107,24 @@ const WEBHOOK_COLUMNS = [
     type: 'Text',
     label: t('Status'),
   },
+  {
+    id: 'vt_webhook_fc11',
```

Comment:
In the meantime, the way we create virtual ids have change (sorry for that :( ). Can you replace this line as in suggestion?
```suggestion
    id: VirtualId(),
```